contrib/python-chess/template.py

```diff
@@ -0,0 +1,32 @@
+pkgname = "python-chess"
+pkgver = "1.10.0"
+pkgrel = 0
+hostmakedepends = [
+    "python-setuptools",
+]
+depends = ["python"]
+checkdepends = ["stockfish"]
+pkgdesc = "Pure Python chess library with move generation and validation"
+maintainer = "shtayerc <david.murko@mailbox.org>"
+license = "GPL-3.0-or-later"
+url = "https://github.com/niklasf/python-chess"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "832fa7d589ffe916f5e84f08ece0d634dfb50568867932e525edcebb573041df"
+
+
+def do_check(self):
+    self.do("python", "setup.py", "test")
+
+
+def do_build(self):
+    self.do("python", "setup.py", "build")
+
+
+def do_install(self):
+    self.do(
+        "python",
+        "setup.py",
+        "install",
+        "--prefix=/usr",
+        f"--root={self.chroot_destdir}",
+    )
```

Comment:
this should use pep_517 like the others, and it works fine. things don't need a pyproject.toml to use it, it uses setuptools normally

Reply:
I could not get default pep517 check working. It does not detect any test file and then fails the build.

Reply:
setup.py test is fine here because that just invokes `python -m unittest -v`. i meant the rest of the stuff